v1alpha1/meshmodel.proto

```diff
@@ -229,11 +260,225 @@ message Resilience {
     TimeLimiter timeLimiter = 4; 
 }
 
-//message Canary {
-//    repeat CanaryRules canaryRule = 1;
-//}
+// Canary configures rules to implement canary deployment. It belongs to the traffic scheduling domain.
+// The developer can deploy a canary version of microservice and accept coloring test its stability before
+// rolling out to the whole instances. 
+message Canary {
+    // CanaryRules is the mesh service's all rules for canary deployment. 
+    repeated CanaryRule canaryRules = 1;
+}
+
+// Sidecar configures the sidecar for every mesh service instances. It works inside the same
+// pod with the workload instance. 
+message Sidecar {
+    // DiscoveryType configures the type of service register/discovery type, its value are among 
+    // "eureka","consul", and "nacos".
+    string discoveryType = 1;
+
+    // Address is the registry center address for workload to visit. 
+	string address = 2; 
+
+    // IngressPort is the port for ingress traffic.
+	int32 ingressPort = 3;
+
+    // IngressProtocol is the protocol for ingress traffic. Its value is "http".
+	string ingressProtocol = 4;
+
+    // egressPort is the port for egress traffic.
+	int32 egressPort = 5;
+
+    // egressProtocol is the protocol for egress traffic. Its value is "http"
```

Comment:
1. Align indentation of comment to code
2. Comment first word should be same with code